components/post_view/post_header/index.js

```diff
@@ -3,15 +3,17 @@
 
 import {connect} from 'react-redux';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {getUser} from 'mattermost-redux/selectors/entities/users';
 
 import PostHeader from './post_header.jsx';
 
-function mapStateToProps(state) {
+function mapStateToProps(state, ownProps) {
     const config = getConfig(state);
     const enablePostUsernameOverride = config.EnablePostUsernameOverride === 'true';
 
     return {
         enablePostUsernameOverride,
+        user: getUser(state, ownProps.post.user_id),
```

Comment:
We prevented to pass user object before on post header and its child components.  Can we just pass a boolean value like userIsBot?

Reply:
Yes, that is possible. Will fix.